Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -971,12 +975,45 @@ private ColumnFamilyHandle GetColumnFamily(RocksDb db, Guid? chainId = null)
             }
             catch (KeyNotFoundException)
             {
-                cf = db.CreateColumnFamily(new ColumnFamilyOptions(), cfName);
+                cf = db.CreateColumnFamily(_options, cfName);
             }
 
             return cf;
         }
 
+        private ColumnFamilies GetColumnFamilies(DbOptions options, string dbName)
+        {
+            var dbPath = Path.Combine(_path, dbName);
+            var columnFamilies = new ColumnFamilies();
+            List<string> listColumnFamilies;
+
+            try
+            {
+                listColumnFamilies = RocksDb.ListColumnFamilies(options, dbPath).ToList();
+            }
+            catch (RocksDbException)
+            {
+                listColumnFamilies = new List<string>();
+            }
+
+            foreach (string name in listColumnFamilies)
+            {
+                columnFamilies.Add(name, _options);
+            }
+
+            return columnFamilies;
+        }
+
+        private RocksDb RocksDbOpen(
```

Comment:
Trivial, but we prefer this style...

```suggestion
        private RocksDb OpenRocksDb(
```

Reply:
Right, changed. 23555ae